### PR TITLE
minor fixes 20231211

### PR DESCRIPTION
**Pull request by winglian**

### CODE REVIEW

File: src/axolotl/utils/models.py [hunk 1]
```diff
@@ -191,6 +191,7 @@ def load_model(
 
     # TODO refactor as a kwarg
```
Comment (NanoCode012) [inline src/axolotl/utils/models.py:192]: there's this todo here.

**Review by NanoCode012 — COMMENTED** [review]
Could we also add `nano` in base docker :)

File: src/axolotl/utils/models.py [hunk 2]
```diff
@@ -535,7 +536,7 @@ def load_model(
 
     model, lora_config = load_adapter(model, cfg, cfg.adapter)
 
-    if cfg.ddp and not load_in_8bit:
+    if cfg.ddp and not load_in_8bit and not load_in_4bit:
         model.to(f"cuda:{cfg.local_rank}")
```
Comment (NanoCode012) [inline src/axolotl/utils/models.py:540]: what's the side effect of this change? Does it mean, the models will all live on gpu0 for 4 bit now?

Comment (winglian) [reply src/axolotl/utils/models.py:540]: there's an edge case that I had to have this set when doing DPO qlora for Mixstral. maybe it was a deepspeed thing and I should change this?

File: src/axolotl/utils/models.py (hunk 1 above)
Comment (winglian) [reply src/axolotl/utils/models.py:192]: I'm not even sure what this TODO is for atm

Comment (winglian) [reply src/axolotl/utils/models.py:192]: maybe it's for the `.from_pretrained` call where these are kwargs for that? should we add this to model_kwargs and remove those from all the from_pretrained calls?

Comment (NanoCode012) [reply src/axolotl/utils/models.py:192]: This is just above

```
   load_in_8bit = cfg.load_in_8bit
     load_in_4bit = cfg.load_in_4bit
```

**Comment by NanoCode012** [comment]
I found that you separated your apt installs into 2 separate docker file. Maybe, these should also be moved to base

https://github.com/OpenAccess-AI-Collective/axolotl/blob/7fabc4d95e9b34e0cfaace2a497cd4fedc43db3b/docker/Dockerfile#L13C1-L13C32

**Pull request closed without merging**